src/couch_httpd.erl

```diff
@@ -761,13 +776,14 @@ send_json(Req, Code, Headers, Value) ->
     AllHeaders = add_headers(Req, maybe_add_default_headers(Req, Headers)),
     send_response(Req, Code, AllHeaders, {json, Value}).
 
-add_headers(#httpd{mochi_req=MochiReq}=Req, Headers) ->
-    Headers1 = http_1_0_keep_alive(MochiReq, Headers),
-
-    Headers2 = Headers1 ++ server_header() ++
-               couch_httpd_auth:cookie_auth_header(Req, Headers1),
+add_headers(Req, Headers0) ->
```

Comment:
You are changing the order of headers here. This is alright. But please make sure that all tests pass. I remember seeing few tests which do not use `?assertEqual(lists:usort(Expected), lists:usort(Headers))`. But rely on literal equivalence (`?assertEqual(Expected, Headers)`).